dls-controls/malcolmjs
# Feature/malcolm refs #279

## Description

Moved the title of the site to be loaded from settings

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Update your settings file with `title: PANDA`, start the site, note that it starts with "Loading..." and then switches to "PANDA v1.2.3"

## Agile board tracking

connect to #279 
closes #279
